english/daria/daria.md

```diff
@@ -0,0 +1 @@
+You decide you need to laugh.  Do you [watch Daria](https://www.youtube.com/watch?v=2UstSviOcyE) or do you [watch Will and Grace](https://www.youtube.com/watch?v=dtX6xXbegdY&list=PLbYaOVNThENwXit5Vg0xZGKlwbjBGUJa2)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.